lessons/Algebra2/exploring-linear-models/langs/en-us/index.adoc

```diff
@@ -0,0 +1,265 @@
+
+=== Launch
+
+[.lesson-instruction]
+- Today we'll be looking at data from each state in the US, and searching for possible relationships within the data!
+- Open the @starter-file{alg2-states}
+- From the File menu, select "Save a Copy", and click "Run".
+- Working in pairs or small groups, complete questions 1 and 2 in @printable-exercise{exploring-states.adoc}.
+
+Have students share back what they Notice and Wonder about the dataset.
+
+=== Investigate
+
+The code on lines 13 and 14 show us how Pyret defines values:
+
+```
+alabama = row-n(states-table, 0)
+alaska  = row-n(states-table, 1)
+```
+
+In math, `x = 4` will define a variable `x` to be the value `4`. Any time we use `x` after that, we can substitute in the value of `4`. This works in Pyret too! But in Pyret, values can be more than just numbers! On lines 13 and 14 of this file, the variables `alabama` and `alaska` are _rows_ from the table.
+
+[.lesson-instruction]
+- Working in pairs or small groups, complete questions 3 and 4 (making displays and suggesting possible relationships) in @printable-exercise{exploring-states.adoc}.
+- **Do not go on to question 5!** We'll be returning to that later in the lesson.
+
+Note: During this activity, don't let students just make scatter plot for random pairs of columns. Instead, encourage them to first _think about which columns might be related_, and then create the scatter plot to search for this relationship. The dataset is designed so that students will quickly begin searching for relationships between varying levels of education and income, and there are linear relationships in each of these.
+
+=== Synthesize
+
+- Share your scatter-plots with one another (copying and pasting scatter plot displays into a shared document, for example, and then labeling those displays). What possible relationships did you find in question 3?
+- Did you and your classmates commonly use any words to describe the relationships you observed?
+** Teacher note: the next section of the lesson is all about identifying form, direction, and strength. Students will acquire the formal vocabulary that data scientists use to assess relationships.
+
+== Describing Patterns @duration{15 minutes}
+
+=== Overview
+Students identify and make use of correlations in scatter plots, learning to characterize their form as being linear, curved, or showing no clear pattern. They learn how to describe the @vocab{strength} of correlations, and that linear patterns have direction.
+
+=== Launch
+
+Scatter plots let us visualize the relationship between two columns. If no relationship exists, the points in the scatter plot just appear as a shapeless cloud. But if there _is_ a relationship, the points will form some kind of pattern. When we build scatter plots, we are searching for patterns - or @vocab{correlations} between two quantitative variables.
+
+These patterns can be described using three qualities: form, direction, and strength.
+
```

Comment:
There is a lot of information here that students don't have access to. I suggest making a one pager

Reply:
Fixed